sympy/printing/pretty/pretty.py

```diff
@@ -1522,6 +1521,30 @@ def _print_Contains(self, e):
         else:
             return prettyForm(sstr(e))
 
+    def _print_SeqFormula(self, s):
```

Comment:
But no one should ever be creating instances of `SeqExpr` - that's just a base class. Not handling that case is fine, as it should never occur (correct me if I'm wrong).